core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -203,9 +203,24 @@ class Partition(val topic: String,
     allReplicasMap.remove(replicaId)
   }
 
-  def removeFutureLocalReplica() {
+  def futureReplicaDirChanged(newDestinationDir: String): Boolean = {
```

Comment:
Could this be private?

Reply:
Thanks for the review @hachikuji. This can not made private because it is used in `ReplicaManager.alterReplicaLogDirs()`.